KVKCalendar/Classes/Style.swift

```diff
@@ -102,6 +102,7 @@ public struct TimelineStyle {
     public var eventCorners: UIRectCorner = .allCorners
     public var eventCornersRadius: CGSize = CGSize(width: 5, height: 5)
     public var minimumPressDuration: TimeInterval = 0.5
+    public var currentLineHourTimeZone: TimeZone = TimeZone.current
```

Comment:
Why didn't you use a property `timezone` from `Style`?

Reply:
It wasn't setting the TimeZone for the CurrentLine hour, the line hour is using Date() to get it's time and style isn't applied. Or you're suggesting that instead of creating a new variable currentLineHourTimeZone I should just use the already existing property timezone from style?

Reply:
correct, we may to try to use existing property timezone from style 🤔

Reply:
style.timezone is used in only one case in getDaysInMonth in YearData Struct. I've changed it. let me test it, I'm currently only using Month and Day type calendar in my project.